Arrowgene.Ddon.LoginServer/Handler/ClientLoginHandler.cs

```diff
@@ -91,7 +91,7 @@ public override void Handle(LoginClient client, StructurePacket<C2LLoginReq> pac
                 }
 
                 TimeSpan loginTokenAge = account.LoginTokenCreated.Value - now;
-                if (loginTokenAge > TimeSpan.FromDays(1)) // TODO convert to setting
+                if (loginTokenAge > TimeSpan.FromDays(7)) // TODO convert to setting
```

Comment:
Ideally this should be a property in settings thought, however it might be a bit annoying as it requires everyone to update the properties files as there is no migration strategy or fallback system, meaning that if it is not defined/updated the default value of the prop would be 0.

with that said I'm okay with this change to not break alot of installations for now, perhaps we can have a list of breaking changes todo and if there are more we can batch them up as a single update.

Just some thoughts here

Reply:
Since I'm still in the learning process I lack the knowledge to convert this to the settings and like you said, can break more things if I try so I prefer to leave this for someone more skilled for now, I only changed this thinking that it could be an issue if the token is expired while someone is still ingame since 24hrs is easily reached by some players.